testsuite/MDAnalysisTests/analysis/test_pca.py

```diff
@@ -199,7 +199,7 @@ def test_project_compare_projections(u_fresh):
     coord0 = project0(u_fresh.trajectory.ts).positions
     u_fresh.trajectory[0]
     coord1 = project1(u_fresh.trajectory.ts).positions
-    assert not np.allclose(coord0, coord1, rtol=1e-05)
+    not np.allclose(coord0, coord1, rtol=1e-05)
```

Comment:
There should be a test here. Probably `assert_allclose` has no not parameter and therefore we should stick with the current test.

Reply:
This is still not testing!